app/src/lib/get-os.ts

```diff
@@ -46,6 +46,14 @@ export const isMacOSVentura = memoizeOne(
     systemVersionLessThan('14.0')
 )
 
+/** We're currently running macOS and it is macOS Ventura. */
+export const isMacOSSonoma = memoizeOne(
+  () =>
+    __DARWIN__ &&
+    systemVersionGreaterThanOrEqualTo('14.0') &&
+    systemVersionLessThan('15.0')
```

Comment:
Sadly, this either means version 15 will need a similar addition or they will fix it in 15.. They did make some improvements from 13 to 14. 🤞